tests/Agent/UnitTests/Core.UnitTest/WireModels/TransactionEventWireModelTests.cs

```diff
@@ -30,7 +30,8 @@ private static IConfiguration CreateMockConfiguration()
                 Mock.Arrange(() => configuration.CrossApplicationTracingEnabled).Returns(true);
                 //Mock.Arrange(() => configuration.CaptureAttributesExcludes)
                 //    .Returns(new List<string>() { "identity.*", "request.headers.*", "response.headers.*" });
-                Mock.Arrange(() => configuration.CaptureRequestParameters).Returns(true);
+                Mock.Arrange(() => configuration.CaptureAttributesIncludes).Returns(new string[] { "request.parameters.*" });
```

Comment:
Is there a reason why we need to set this ? Tests in this files seems to pass without this line of code.